### PR TITLE
Finished ticket 5

### PR DESCRIPTION
**Pull request by 1991hannah**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -76,6 +76,29 @@ describe('GET /api/articles/:article_id', () => {
...
+                    author:expect.any(String),
+                    created_at:expect.any(String),
+                    votes:expect.any(Number),
+                    article_img_url:expect.any(String),
+                    comment_count:expect.any(String)
+                })
+            })
+        })
+    })
+})
```
Comment (NRMSMITH) [inline __tests__/app.test.js:108]: Gorgeous testing - I want one more test to check that the articles are sorted by date in desc order! Have a look at jest-sorted for this! :D

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
One test to complete :D (your code already does it but we need to prove it with the test!)

File: __tests__/app.test.js (hunk 1 above)
Comment (1991hannah) [reply __tests__/app.test.js:108]: thanks Niamh! Have added and pushed the test up :)

Comment (1991hannah) [reply __tests__/app.test.js:108]: Am I ok to merge the pull request?

**Pull request merged**